### DISCUSSION TITLE
Modals only with reply? other ways in the future?

### DISCUSSION THREAD

**Discussion by michel929** [discord-jda/JDA#2367] [Questions and Help] [answered]
Will there ever be something like sendModal(User u, Modal m) or is it intended that there is only replyModal()...

**Comment by MinnDevelopment** [comment, marked as answer]
Discord has no plans to add other ways to reply with modals at this time. Keep in mind that opening modals at random would be a pretty bad user experience. If you want to open a modal later, you should use a button.

**Comment by michel929** [comment]
Ok thx :)
